src/components/CommentViewer.jsx

```diff
@@ -14,18 +15,28 @@ const Wrapper = styled.div`
 `;
 
 function CommentViewer({
-  userId, comments, onClickEdit, onClickDelete,
+  comments, onClickEdit, onClickDelete,
 }) {
+  const [uid, setUid] = useState(null);
+
+  useEffect(() => {
+    const user = firebase.auth().currentUser;
+
+    if (user) {
+      setUid(user.uid);
+    }
+  }, [firebase.auth().currentUser]);
```

Comment:
firebas.auth().currentUser는 객체이기 때문에 렌더링 될 때마다 매번 실행될 것 같습니다. 

```suggestion
 const user = firebase.auth().currentUser;
 
   useEffect(() => {
    if (user) {
      setUid(user.uid);
    }
  }, [user.uid]);
```
참조링크: https://sgwanlee.medium.com/useeffect%EC%9D%98-dependency-array-ebd15f35403a

Reply:
앗 그렇겠네요! 확인해주셔서 감사합니다 👍